sqlmesh/core/engine_adapter/snowflake.py

```diff
@@ -280,6 +292,11 @@ def _get_data_objects(
 
         schema = to_schema(schema_name)
         catalog_name = schema.catalog or self.get_current_catalog()
+        if catalog_name:
+            catalog_name = normalize_identifiers(catalog_name, dialect=self.dialect).sql(
```

Comment:
is this problematic if the schema.catalog is duckdb?

Reply:
In trying to answer your question, I realised this was no longer needed. The tests pass without it, which means I probably added it to get a certain test to pass and then made a wider change to get other tests to pass which made this redundant.

I've removed it